src/ai/backend/manager/idle.py

```diff
@@ -66,6 +67,7 @@ class AppStreamingStatus(enum.Enum):
 class BaseIdleChecker(aobject, metaclass=ABCMeta):
 
     name: ClassVar[str] = "base"
+    check_period: float = 10.0
 
     _db: SAEngine
```

Comment:
Let's update the type annotation here: `ai.backend.manager.models.utils.ExtendedAsyncSAEngine`.